db/re/item_db.txt

```diff
@@ -4030,7 +4030,14 @@
 5894,RCC2013_3RD_CROWN,RCC2013 3RD CROWN,4,20,,2500,,7,,1,0xFFFFFFFF,63,2,256,,1,0,1108,{ bonus bAllStats,3; skill "AL_TELEPORT",1; },{ sc_start SC_SPEEDUP0,-1,25; },{ sc_end SC_SPEEDUP0; }
 5895,RCC2013_ANV_Hat,RCC2013 ANV Hat,4,20,,2500,,7,,1,0xFFFFFFFF,63,2,256,,1,0,1109,{ bonus bAllStats,1; bonus2 bAddClass,Class_All,2; bonus bMatkRate,2; },{},{}
+5900,Divine_Guard_Hat,Divine Guard Hat,4,0,,200,,10,,1,0x00004000,63,2,256,,100,1,1121,{ .@r = getrefine(); bonus2 bSubSize,Size_All,5; if(.@r>=7) { bonus3 bAutoSpell,"AL_HEAL",max(getskilllv("AL_HEAL"),5),10; } if(.@r>=9) { bonus2 bSkillAtk,"LG_RAYOFGENESIS",20; } },{},{}
+5901,Focus_Beret,Focus Beret,4,0,,200,,10,,1,0x00008000,63,2,256,,100,1,1122,{ .@r = getrefine(); bonus4 bAutoSpellOnSkill,"SR_DRAGONCOMBO","SR_FALLENEMPIRE",1,10; if(.@r>=7) { autobonus "{ bonus bAspd,2; }",30,5000,0,"{ specialeffect2 EF_ENHANCE; }"; } if(.@r>=9) { bonus2 bSkillAtk,"SR_FALLENEMPIRE",30; } },{},{}
+5902,Harvester_Hat,Harvester Hat,4,0,,200,,10,,1,0x00040000,63,2,256,,100,1,1123,{ .@r = getrefine(); bonus2 bAddMonsterDropItem,6211,100; bonus2 bAddMonsterDropItem,6212,100; if(.@r>=7) { bonus2 bAddMonsterDropItem,6210,50; } if(.@r>=9) { bonus2 bSkillAtk,"GN_CARTCANNON",15; } },{},{}
+5903,Deadman_Bandana,Deadman Bandana,4,0,,200,,15,,1,0x00810204,63,2,256,,99,1,1124,{ .@r = getrefine(); bonus bCritical,.@r*2; bonus bFixedCastrate,-2; bonus3 bAutoSpell,"SL_SWOO",max(getskilllv("SL_SWOO")),10; if(.@r>=7) { bonus bMatk,20; bonus2 bCastrate,"SL_SWOO",3; } if(.@r>=9) { bonus bVariableCastrate,-20; bonus2 bSkillUseSP,"SL_STUN",5; bonus2 bSkillUseSP,"SL_SMA",5; } },{},{}
+5904,Inconspicuous_Hat,Inconspicuous Hat,4,0,,200,,10,,1,0x00020000,63,2,256,,100,1,1125,{ .@r = getrefine(); if(.@r<9) { bonus bMatkRate,7; } else if(.@r>=9) { bonus bMatkRate,12; } if(.@r>=7) { bonus bMatk,readparam(bInt)/5*2; } },{},{}
+5905,Lyrica_Hat,Lyrica Hat,4,0,,200,,10,,1,0x00080000,63,2,256,,100,1,1126,{ .@r = getrefine(); bonus2 bSkillAtk,"WM_SEVERE_RAINSTORM_MELEE",15; if(.@r>=7) { bonus bFixedCastrate,4*max(getskilllv("WM_LESSON")); } if(.@r>=9) { bonus2 bSkillUseSP,"WM_SEVERE_RAINSTORM_MELEE",10; } },{},{}
 5906,Oni_Horns_,Oni Horns,4,0,,20,,10,,1,0x00001000,63,2,256,,100,1,1127,{ bonus2 bSkillAtk,"GC_COUNTERSLASH",(10*getskilllv("GC_WEAPONBLOCKING")); .@r = getrefine(); if(.@r > 6){ bonus bAtk,20; bonus bHit,5; } if(.@r > 9){ bonus3 bAutoSpell,"GC_CROSSIMPACT",1,10; } },{},{}
+5907,Sea_Captain_Hat,Sea Captain Hat,4,0,,200,,10,,1,0x00000400,63,2,256,,100,1,1128,{ .@r = getrefine(); bonus2 bSkillAtk,"NC_AXEBOOMERANG",20; if(.@r<7) { bonus2 bSkillAtk,"NC_AXETORNADO",20; } else if(.@r>=7) { bonus2 bSkillAtk,"NC_AXETORNADO",max(getskilllv("BS_WEAPONRESEARCH")+20); } else if(.@r>=9) { bonus2 bSkillAtk,"NC_AXETORNADO",max(getskilllv("BS_WEAPONRESEARCH")+10); bonus2 bSkillAtk,"NC_AXETORNADO",max(getskilllv("NC_TRAININGAXE")+10); } },{},{}
```

Comment:
https://www.divine-pride.net/database/item/5907
Class **56**
`This item cannot be traded with other accounts. ` -> item_trade.txt
The bonus can be write
```
bonus2 bSkillAtk,"NC_AXEBOOMERANG",20;
.@dmg = 20;
.@r = getrefine();
if (.@r>=7) { .@dmg += getskilllv("BS_WEAPONRESEARCH"); }
if (.@r>=9) { .@dmg += getskilllv("NC_TRAININGAXE"); }
bonus2 bSkillAtk,"NC_AXETORNADO",.@dmg;
```